api/v1beta1/events.go

```diff
@@ -185,79 +193,97 @@ var Events = map[DisruptionEventReason]DisruptionEvent{
 	},
 	EventDisruptionCreationFailed: {
 		Type:                        corev1.EventTypeWarning,
 		Reason:                      EventDisruptionCreationFailed,
 		OnDisruptionTemplateMessage: "Injection pod for disruption \"%s\" failed to be created",
-		Category:                    DisruptEvent,
+		Category:                    DisruptionEvent,
 	},
 	EventDisruptionStuckOnRemoval: {
 		Type:                        corev1.EventTypeWarning,
 		Reason:                      EventDisruptionStuckOnRemoval,
 		OnDisruptionTemplateMessage: "Instance is stuck on removal because of chaos pods not being able to terminate correctly, please check pods logs before manually removing their finalizer. https://github.com/DataDog/chaos-controller/blob/main/docs/faq.md",
-		Category:                    DisruptEvent,
+		Category:                    DisruptionEvent,
 	},
 	EventInvalidDisruptionLabelSelector: {
 		Type:                        corev1.EventTypeWarning,
 		Reason:                      EventInvalidDisruptionLabelSelector,
 		OnDisruptionTemplateMessage: "%s. No targets will be selected.",
-		Category:                    DisruptEvent,
+		Category:                    DisruptionEvent,
 	},
 	EventDisruptionNoMoreValidTargets: {
 		Type:                        corev1.EventTypeNormal,
 		Reason:                      EventDisruptionNoMoreValidTargets,
 		OnDisruptionTemplateMessage: "No more targets found for injection for this disruption (either ignored or already targeted by another disruption)",
-		Category:                    DisruptEvent,
+		Category:                    DisruptionEvent,
 	},
 	EventDisruptionNoTargetsFound: {
 		Type:                        corev1.EventTypeWarning,
 		Reason:                      EventDisruptionNoTargetsFound,
 		OnDisruptionTemplateMessage: "The given label selector did not return any targets. Please ensure that both the selector and the count are correct (should be either a percentage or an integer greater than 0).",
-		Category:                    DisruptEvent,
+		Category:                    DisruptionEvent,
 	},
 	EventInvalidSpecDisruption: {
 		Type:                        corev1.EventTypeWarning,
 		Reason:                      EventInvalidSpecDisruption,
 		OnDisruptionTemplateMessage: "%s",
-		Category:                    DisruptEvent,
+		Category:                    DisruptionEvent,
 	},
 	EventDisruptionChaosPodCreated: {
 		Type:                        corev1.EventTypeNormal,
 		Reason:                      EventDisruptionChaosPodCreated,
 		OnDisruptionTemplateMessage: "Created disruption injection pod for \"%s\"",
-		Category:                    DisruptEvent,
+		Category:                    DisruptionEvent,
 	},
 	EventDisruptionCreated: {
 		Type:                        corev1.EventTypeNormal,
 		Reason:                      EventDisruptionCreated,
-		OnDisruptionTemplateMessage: "Disruption created",
-		Category:                    DisruptEvent,
+		OnDisruptionTemplateMessage: "DisruptionEvent created",
```

Comment:
I think this block of event messages got caught up in a string replace? These should look like
Disruption Created
DisruptionCron created
DisruptionCron deleted
and so on,
not `DisruptionEvent cron updated`